### ISSUE TITLE
MIT License

### ISSUE THREAD

**Issue by ValentinKlinghammer** [victorzimmer/MapKit#15]
Hey there,

thanks for your great work! I'm currently working on a fork of this repo and would love to continue with it. However, you have chosen to create [your own license](http://hotdot.no/hotdot_open-source_license.txt) which is, at the time of writing, not available to read (404). 

Personally, I feel it would be great if you'd just switch to a common license for Open Source projects, MIT seems a great choice.

Currently, I have really no idea wether I am allowed to do something with the code or not 😅

**Comment by victorzimmer** [comment]
Absolutely. No idea what happened to the Web Server, but the license was originally based upon the MIT license if I'm not wrong. I'll upload the file to the repository later today!

In short though, you can do whatever you'd like with it :)

**Comment by ValentinKlinghammer** [comment]
Awesome, thanks for clarification.

You did great work, thanks again!
